mteb/tasks/Retrieval/zho/NeuCLIR2023ZhoRetrieval.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class NeuCLIR2023ZhoRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="NeuCLIRZho2023",
```

Comment:
What is the difference between 2023 and 2022? any reason to not just use 2023?

Reply:
Each one is a different evaluation set for that year's shared task (which is what a TREC track is)

Reply:
If we think there's too much NeuCLIR we can remove one of the years, but they are distinct

Reply:
Thanks for letting me know. Seem like the organizers -- at least as I understand it in the [docs](https://neuclir.github.io/2023#data) -- combined 2022 into 2023 (at least partly, they used it as the dev set). This seems to suggest to me that the datasets are similar enough to warrant using only the datasets. 

@imenelydiaker what are your thoughts on this?

Reply:
I don't think the docs say that- do you mind posting the quote?  

The queries and qrels are different as I said above; the collection is shared but so are retrieval datasets that use Wikipedia (FEVER and NQ both) or MSMarco (TREC DL 19, TREC DL 20, etc.).  As someone who is advised by the creators of this dataset and who uploaded it, I can promise you the datasets are different :) 

I can see an argument for too much data of one style though! If we want less of it, I can understand that.

Alternatively though MMTEB (and MTEB) already has a lot of Wikipedia-style automatically generated data and could benefit from TREC-style data (which is human judged rather than automatically created and has much deeper relevance judgements).

Reply:
> I don't think the docs say they are the same- do you mind posting the quote? They do say they you can use it as a dev set if you want to get better performance on the 2023 version but they are distinct datasets -- no more than I would say FEVER is a dev set just because training on it makes you do better on a different fact checking dataset.

At least as I understand it; NeuCLIR 2023 includes two dev sets one of which is the 2022 eval set:

> The second development set comprises the NeuCLIR 2022 evaluation set.

But I agree with your point. The two splits you compare are not the same. I am sorry if it came across that way.

> I think the biggest reason to not put them as a single dataset is that there are published papers reporting scores on each of them separately. Here are [papers reporting results](https://scholar.google.com/scholar?cites=10465691959091815374&as_sdt=20000005&sciodt=0,21&hl=en) for 2022 and different ones for [2023](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C21&q=neuclir+track+2023+trec&btnG=).

I am not arguing that we combine it into one dataset. I am wondering if the two datasets test the same underlying phenomena. If that is the case, we might consider whether it is worth adding both of them in or whether one is fine. The MTEB is already quite large and we want to make sure that we make the benchmark unpractical to work with. 

I am atm. leaning toward including both datasets. If the datasets is then later revealed to be highly correlated we might consider not adding both datasets to the benchmark list, but I believe that is better to tested empirically.

Reply:
Let's roll with two dataset 🚀